spec/vcr_cassettes/manageiq/providers/hawkular/middleware_manager/refresh_parser.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
I had the logic of fetching the deployment availability on the parse_deployment. This won't be needed as I'm moving this. (I'll delete this cassette)